### ISSUE TITLE
Update production models without reprocessing all data

### ISSUE THREAD

**Issue by leonardoobaptistaa** [ankane/eps#10]
Hi there

I am using EPS version 0.1.1 on production to predict some values to detect fraudulent data.
The algorithm itself works fine, but I am struggling to update the models with new values.

I have the models saved on database, exported to JSON (I know it's not supported on newer versions, I am planning to migrate soon) and they are loaded (Regressor.load_json) when I need to make a prediction.

But now, I need to update these models daily and my training data is too big and takes a lot of time to normalize data, remove outliers, etc..
Is there a way to load a stored model, add new data, retrain and update the model with just the new data (without loading all previous trained data) ?

I thought the Daru Dataframes could be used to achieve this, but it seems that is not the case (or I can't find how).

If there isn't a way to do that with EPS (no matter what version), someone know if it can be done with any other Python framework?

Thanks!

**Comment by ankane** [comment]
Hey @leonardoobaptistaa, it sounds like you're looking for a model to do online learning. Eps doesn't support this, but a library like xLearn can (has [Ruby](https://github.com/ankane/xlearn) and [Python](https://github.com/aksnzhy/xlearn) bindings). Also, some Scikit-learn models have a `partial_fit` method for this.

**Comment by ankane** [comment]
Vowpal Wabbit also supports this ([Ruby](https://github.com/ankane/vowpalwabbit) and [Python](https://github.com/VowpalWabbit/vowpal_wabbit)).

**Comment by leonardoobaptistaa** [comment]
Hi @ankane 

It looks like Vowpal Wabbit has very similar methods, I will try to use it instead of EPS.
Thanks for pointing me in the right direction here.

And thanks for these project too, they make classification/prediction really easy for Ruby devs.
Have a great day, bye
